reach2

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+
+case "$1" in
+  "bootstrap") echo "TODO" ;; # Document this in `optparse` as well
```

Comment:
I'm reading your mind and thinking that you shouldn't have this... the normal path should just check to see if the Docker image is around and download it if it isn't around